packages/api/.env.default

```diff
@@ -21,33 +21,41 @@ JWT_RESET_SECRET=Production_is_secured_with_a_long_random_string
 JWT_FARM_SECRET=Here_we_can_use_friendly_explanations
 JWT_SCHEDULER_SECRET=Another_token_was_needed_for_the_scheduler
 
-OPEN_WEATHER_APP_ID=?
+# Create your own (free in most cases) Google API key at https://console.cloud.google.com/apis/dashboard
+# Same as the VITE_GOOGLE_MAPS_API_KEY under webapp
 GOOGLE_API_KEY=?
 
-GMAIL_API_CLIENT_ID=?
-GMAIL_API_CLIENT_SECRET=?
-GMAIL_ACCESS_TOKEN=?
-GMAIL_REFRESH_TOKEN=?
+# Create your own free or student OpenWeather API key at https://openweathermap.org/price
+# Same as the VITE_WEATHER_API_KEY under webapp
+OPEN_WEATHER_APP_ID=?
```

Comment:
Were there any limitations on this public key? I thought someone mentioned that there might be for some features?